packages/diagnostics_channel/src/index.js

```diff
@@ -0,0 +1,39 @@
+'use strict'
+
+const {
+  Channel,
+  channel
+} = require('diagnostics_channel') // eslint-disable-line n/no-restricted-require
+
+const [major, minor] = process.versions.node.split('.')
+
+// Our own DC with a limited subset of functionality stable across Node versions.
+// TODO: Move the rest of the polyfill here.
+// TODO: Switch to using global subscribe/unsubscribe/hasSubscribers.
+const dc = { channel }
+
+// Prevent going to 0 subscribers to avoid bug in Node.
+// See https://github.com/nodejs/node/pull/47520
+if (major === '19' && minor === '9') {
+  const channel = dc.channel
```

Comment:
Is this line needed? I believe that this point `channel === dc.channel`, and reassigning `dc.channel` on line 20 would still leave `channel` available on line 21.